src/Features/LanguageServer/Microsoft.CommonLanguageServerProtocol.Framework/RequestExecutionQueue.cs

```diff
@@ -49,7 +52,7 @@ namespace Microsoft.CommonLanguageServerProtocol.Framework;
 public class RequestExecutionQueue<TRequestContext> : IRequestExecutionQueue<TRequestContext>
 {
     protected readonly ILspLogger _logger;
-    private readonly IHandlerProvider _handlerProvider;
+    protected readonly IHandlerProvider _handlerProvider;
```

Comment:
rename.  protected members shoudl be PascalCased

Reply:
I'm just following the existing pattern in this file. There are other protected fields already: _logger and _queueProcessingTask.

Reply:
ok.  can leave this for now.  i can fix up when this goes in.  def not a normal naming stype :)